lambda/executor.js

```diff
@@ -168,10 +191,18 @@ const runInSeries = async({num, lambdaARN, lambdaAlias, payloads, preARN, postAR
     return results;
 };
 
-const computeStatistics = (baseCost, results, value, discardTopBottom) => {
+const computeStatistics = (baseCost, results, value, discardTopBottom, onlyColdStarts) => {
     // use results (which include logs) to compute average duration ...
 
     const durations = utils.parseLogAndExtractDurations(results);
+    if (onlyColdStarts) {
+        // we care about the init duration in this case
+        const initDurations = utils.parseLogAndExtractInitDurations(results);
```

Comment:
I think there's a problem with this. The cost is derived from the `durations` array, now that we're adding the `initDuration` in case of cold start the cost calculation will be inaccurate.

Reply:
I changed the way the duration and the cost are calculated. 
Please take a look when you have a moment, there are some Tests missing (planning to add early next week), but should be good to go if you're happy with this.